action/action.go

```diff
@@ -30,7 +30,7 @@ type (
 	actionPayload interface {
 		Cost() (*big.Int, error)
 		IntrinsicGas() (uint64, error)
-		SetEnvelopeContext(SealedEnvelope)
+		SetAbstractAction(AbstractAction)
```

Comment:
same here

Reply:
in `func (elp *envelope) LoadProto()`, it calls: `elp.payload.SetAbstractAction(elp.AbstractAction)`, so `actionPayload ` needs this method